overlord/assertstate/assertstate.go

```diff
@@ -336,7 +344,8 @@ func delayedCrossMgrInit() {
 
 // AutoRefreshAssertions tries to refresh all assertions
 func AutoRefreshAssertions(s *state.State, userID int) error {
-	if err := RefreshSnapDeclarations(s, userID); err != nil {
+	opts := &RefreshSnapDeclarationsOptions{IsAutoRefresh: true}
```

Comment:
do we have a test case for checking that opts is true in this codepath? I see we do for ValidationSetAssertionForMonitor, but I'm unclear if we do for this case

Reply:
Yes, I added an assertion to TestValidationSetAssertionsAutoRefresh. I can add two different tests to ensure that both RefreshSnapDeclarations and RefreshValidationSetAssertions propagate the options